content/6_Advanced/DP_SOS.mdx

```diff
@@ -26,6 +26,85 @@ frequency: 0
 	</Resource>
 </Resources>
 
+
+Sum over subsets (SOS) dp is a trick that allows you to efficiently compute the sum of all the subsets of an array.
+
+The naive solution would be to iterate through every other mask and if the other mask is a subset of the current mask we add it to the current answer.
```

Comment:
```suggestion
The naive solution would be to iterate through every other mask and if the other mask is a subset of the current mask we add it to the current answer.
```

wait this is rly confusing...

clarify?

Reply:
i think it's better now